tests/acceptance/variant_view.js

```diff
@@ -271,17 +190,20 @@ test.describe('Variant View - ss found in both EVA and Accessioning', function()
                 "table", "consequence-type-summary-1", expectedResults, checkSection);
 
     expectedResults = [{"CHROM": "CHROM", "POS": "POS", "ID": "ID", "REF": "REF", "ALT": "ALT"},
-                       {"CHROM": "11", "POS": "50921862", "ID": "CAE11_50921862", "REF": "C", "ALT": "G"}];
+                       {"CHROM": "11", "POS": "50921862", "ID": ".", "REF": "C", "ALT": "G"}];
+    // TODO: Running tests based on table IDs doesn't seem to work because SenchaJS seems to generate multiple tables
+    // with the same ID!!. If this is not remedied, the values in these tests are a moving target
+    // as variants get added to the archive.
```

Comment:
first of all, WAT. 

Secondly, WAT again. I understand nothing about these tests, but:
- is it getting real data from real webservices? if so, why can't you filter the results to check agains the expected object?
- if it's getting the data from stored examples in the code, why can't you do the same?

can't you filter with other fields other than the "non-unique IDs"?

Reply:
I misunderstood this part. the non-unique IDs are for looking through the DOM, so filtering by context is more complicated than I expected